src/api/api.ts

```diff
@@ -0,0 +1,15 @@
+const API_URL_TODOS = 'https://mate.academy/students-api/todos';
+const API_URL_USERS = 'https://mate.academy/students-api/users';
+
+export function getTodos(): Promise<Todo[]> {
+  return fetch(API_URL_TODOS)
+    .then(response => response.json());
+}
+
+export function getUsers(userId: number) {
+  return fetch(`${API_URL_USERS}`)
+    .then(response => response.json())
+    .then(users => {
+      return users.find((user: { id: number; }) => user.id === userId);
+    });
```

Comment:
now you fetch ALL users and then use find to find a single one
is it effective to fetch 1000000000 users each time, for instance, in order to show information of just ONE of them?

let's use a dedicated endpoint to fetch a single user information by it's id
> CurrentUser component receives userId as a prop and loads user details from [GET user endpoint](https://mate.academy/students-api/users/1) (replace 1 with a given userId).